resources/views/site/landingPage/_header----stories.blade.php

```diff
@@ -0,0 +1,84 @@
+@php
+    $newHoursStartAt = Carbon\Carbon::parse('2022-01-02T00:00:00-0600');
```

Comment:
Curious, why does it start at Jan 2nd?

Reply:
@zachgarwood tbh one of those pieces of code that I just copied and pasted lol